es-app/src/views/gamelist/GridGameListView.cpp

```diff
@@ -48,6 +49,30 @@ void GridGameListView::launch(FileData* game)
 	ViewController::get()->launch(game);
 }
 
+void GridGameListView::remove(FileData *game, bool deleteFile)
```

Comment:
In what scenario is the `remove` method needed ?

Reply:
The function "remove" is pure within the class IGameListView. Since GridGameListView inherit from ISimpleGameListView, which inherit from IGameListView, it have to implement this method.

So I had 2 solutions, either copy/paste it from the BasicGameListView, which also inherit from ISimpleGameListView, or move it up in the inheritance tree. I wasn't sure, so I preferred to choose the first solution, but I can change that if needed.